src/chigame/games/views.py

```diff
@@ -68,3 +72,57 @@ def get_context_data(self, **kwargs):
     # check if user is staff member
     def test_func(self):
         return self.request.user.is_staff
+
+
+# Tournaments
+
+
+# Currently, only staff users can create, update, and delete tournaments.
+# This may be changed later if we have an official sets of rules for
+# tournaments creation, update, and deletion.
+
+
+# Permission Checkers
+
+
+def staff_required(view_func):
+    @wraps(view_func)
+    def wrapper(request, *args, **kwargs):
+        if not request.user.is_staff:
+            raise PermissionDenied
+        return view_func(request, *args, **kwargs)
+
+    return wrapper
+
+
+class TournamentDetailView(DetailView):
+    model = Tournament
+    template_name = "tournaments/tournament_detail.html"
+    context_object_name = "tournament"
+
+
+@method_decorator(staff_required, name="dispatch")
+class TournamentCreateView(CreateView):
+    model = Tournament
+    template_name = "tournaments/tournament_create.html"
+    fields = [
+        "name",
+        "game",
+        "start_date",
+        "end_date",
+        "max_players",
+        "description",
+        "rules",
+        "draw_rules",
+        "matches",
+        "num_winner",
```

Comment:
Remove this field as it is causing an error: 
django.core.exceptions.FieldError: Unknown field(s) (num_winner) specified for Tournament

Reply:
Sorry, I forgot to remove the field here